clap_builder/src/builder/possible_value.rs

```diff
@@ -130,6 +133,20 @@ impl PossibleValue {
         self
     }
 
+    /// Add a _visible_ alias to this [PossibleValue].
+    ///
+    /// Unlike [PossibleValue::alias], these aliases will show in help output.
+    #[must_use]
+    pub fn visible_alias(mut self, name: impl IntoResettable<Str>) -> Self {
```

Comment:
This needs a test.  Ideally, the test is added in a prior commit, with a regular alias, and should pass.  In this commit, it would then change to a visible alias with the expected output updated.  

Being this particular makes it very easy to show anyone (1) what is supported today, (2) what this new feature looks like, and (3) how the behavior is different than what is there today.

One of the big design questions that needs discussion in #4416 is what the help output should be like for this.   Note that the design conversation should happen there.